cppForSwig/Makefile

```diff
@@ -40,23 +42,33 @@ else
 PYTHON_INCLUDES=$(shell $(PYVER)-config --includes )
 endif
 
-CPPFLAGS += $(ARMORY_CPPFLAGS) -Icryptopp -Imdb -DUSE_CRYPTOPP -D__STDC_LIMIT_MACROS
-LDLIBS += -lpthread -Lmdb
-SWIG_OPTS    += -c++ -python -threads
-
-SWIG_INC     += 
-
-CXXCPP += $(CPPFLAGS) $(PYTHON_INCLUDES) -std=c++11
+# "-Wl,--no-as-needed" used for Ubuntu 12.04 due to rt library linking issues.
+# http://stackoverflow.com/questions/17150075/undefined-reference-to-clock-gettime-although-lrt-is-given
+# has more info. Require only for static linking. When 12.04 goes, remove this
+# unless it's discovered that OS X needs it.
+ifdef STATIC_LINK
+LDLIBS += -Wl,--no-as-needed
```

Comment:
@droark - Checkout my test branch in my fork (https://github.com/josephbisch/BitcoinArmory/tree/test. I did what theuni suggested by adding a function that calls timer_create to force the linking of librt without --no-as-needed. I put the function in the CppBlockUtils.i swig interface file using %inline. Feel free do do it in a better way if there is one. I don't really know swig.

Why don't you pull it into 0.93-bugfix and then I'll pull it into my gitian branch the exact way you put it into 0.93-bugfix.